modules/gapi/src/compiler/gislandmodel.cpp

```diff
@@ -335,6 +335,24 @@ ade::NodeHandle GIslandModel::producerOf(const ConstGraph &g, ade::NodeHandle &d
     return ade::NodeHandle();
 }
 
+std::string GIslandModel::buildIslandName(const ade::NodeHandle& island_nh, const Graph& g) {
```

Comment:
Also, to avoid confusion with the `GIsland::name()` which already exists and has no relation to this one, I'd rename this to `traceName` or something like that - to reflect the purpose.

Reply:
Thanks! Renamed to `traceIslandName` (added `Island` in the name's middle because function is inside the `GIslandModel` namespace but it is not `GIsland` method).